packages/fluentui/react-component-ref/src/utils.ts

```diff
@@ -42,3 +42,28 @@ export const handleRef = <N>(ref: React.Ref<N> | undefined, node: N) => {
 export const isRefObject = (ref: any): ref is React.RefObject<any> =>
   // https://github.com/facebook/react/blob/v16.8.2/packages/react-reconciler/src/ReactFiberCommitWork.js#L665
   ref !== null && typeof ref === 'object' && ref.hasOwnProperty('current');
+
+/**
+ * Merges up to two React Refs into a single memoized function React Ref so you
+ * can pass it to an element.
+ *
+ * @example
+ * import React from "react";
+ * import { useForkRef } from "reakit-utils";
+ *
+ * const Component = React.forwardRef((props, ref) => {
+ *   const internalRef = React.useRef();
+ *   return <div {...props} ref={useForkRef(internalRef, ref)} />;
+ * });
+ */
+export function useForkRef(refA?: React.Ref<any>, refB?: React.Ref<any>) {
```

Comment:
I don't see any usages of this hook in this PR... is that intended ?

Reply:
There is a usage in `packages/fluentui/react-northstar/src/utils/positioner/Popper.tsx`.

Reply:
FYI, https://github.com/microsoft/fluentui/pull/17276#discussion_r589298186.